test/cql-pytest/cassandra_tests/validation/entities/collections_test.py

```diff
@@ -305,9 +305,9 @@ def testListWithUnsetValues(cql, test_keyspace):
         assert_rows(execute(cql, table, "SELECT l FROM %s WHERE k = 10"), [l])
 
         # select with in clause
-        assert_invalid_message(cql, table, "Invalid unset value for column k", "SELECT * FROM %s WHERE k IN ?", UNSET_VALUE)
```

Comment:
Please don't remove tests - certainly not from cassandra_tests - without a thorough explanation of why. 
"WHERE k in UNSET_VALUE" *should* be an error. Why don't you want it to be an error?
They you moved it below the comment that says that the Python driver catches this case is surprising - did the Python driver change from the version I had to the one you have, and it's now caught the driver?

Reply:
The driver threw an error here on my branch, so I thought that maybe new driver version did some additional checks or something, but after closer investigation it turned out that it was because of my error It passes now without any modifications.